### ISSUE TITLE
Running rec_test.m  without pretraining

### ISSUE THREAD

**Issue by ghost** [zhirongw/3DShapeNets#3]
There are 4 files, discriminative_10_class.mat, discriminative_40_class.mat, generative_10_class.mat and generative_40_class.mat, which I think are pre-trained models. But when I run call rec_test.m with any of these models, I get the following error:

Error using feval
Argument must contain a string or function_handle.

Error in myConvolve2 (line 32)
            target_gpu = feval(kConv,...

Error in rec_test (line 43)
            hidden_presigmoid = myConvolve2(kConv_forward2, batch_data, model.layers{l}.uw, stride,
            'forward'); 

My question is whether pre-training using run_pretrain.m is necessary for running rec_test.m. Also, from where do we get the accuracy of the test data? 

Thanks

**Comment by jawhster** [comment]
I also receive this error when trying to run sample_test_classification, even after running run_pretrain.m and passing in the trained model. Did you ever get an answer to this question? Also, how do I run the model to classify a given depth image (i.e. which file does this)? Thanks, Justin

**Comment by rib4rt** [comment]
Have you received any answers?

**Comment by tr4B** [comment]
Have you received any answers?
